Sources/Internal/FrameProvider.swift

```diff
@@ -178,10 +178,7 @@ final class FrameProvider {
     let origin: CGPoint
     if distanceFromAdjacentDay < 0 {
       let proposedX = adjacentDayFrame.minX - content.horizontalDayMargin - daySize.width
-      if
-        proposedX.alignedToPixel(forScreenWithScale: scale) >=
-          minX.alignedToPixel(forScreenWithScale: scale)
```

Comment:
it was here that 1.49899999 would round down to 1.333333, and 1.5 would round up to 1.6666666, causing the comparison to fail.